integration-test-processor/src/main/java/org/springframework/cloud/stream/module/it/IntegrationTestProcessor.java

```diff
@@ -22,25 +22,18 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
-import org.springframework.cloud.stream.annotation.EnableBinding;
-import org.springframework.cloud.stream.config.ChannelBindingServiceProperties;
-import org.springframework.cloud.stream.messaging.Processor;
 
 /**
  * A module that can misbehave, useful for integration testing of module deployers.
  *
  * @author Eric Bottard
  */
-@EnableBinding(Processor.class)
 @EnableConfigurationProperties(IntegrationTestProcessorProperties.class)
 public class IntegrationTestProcessor {

```

Comment:
Definitely not anymore, no. Note that no test exercised its processor nature before anyway.